app/scripts/modules/titus/package.json

```diff
@@ -2,14 +2,43 @@
   "name": "@spinnaker/titus",
   "license": "Apache-2.0",
   "version": "0.0.175",
-  "main": "lib/lib.js",
-  "typings": "lib/index.d.ts",
+  "main": "dist/titus.module.js",
```

Comment:
We should use `"module": "dist/titus.module.js"` to be a bit more explicit that we're shipping ESM.  https://rollupjs.org/guide/en/#publishing-es-modules

It can be in addition to the `main` field, or we could just replace `main` with `module`.

Reply:
Good call. I missed updating it.